queries/po/highlights.scm

```diff
@@ -0,0 +1,33 @@
+; Keywords
+
+[
+  "msgctxt"
+  "msgid"
+  "msgid_plural"
+  "msgstr"
+  "msgstr_plural"
+] @keyword
+
+; Punctuation
+
+[ "[" "]" ] @punctuation.bracket
+
+; Literals
+
+(string) @string
+
+(escape_sequence) @string.escape
+
+(number) @number
+
+; Comments
+
+(comment) @comment @spell
+
+(comment (reference (text) @string.special.path))
```

Comment:
I'm not keen on captures with three parts to be honest.

```suggestion
(comment (reference (text) @text.uri))
```

Reply:
I don't think that's avoidable, though (especially not with our one-size-fits-none `@text` capture)...

And remember that there's builtin fallbacks, so you don't have to define a separate highlight group for it if you don't want to. There's a reason we only enforce top-level groups in CI.

Reply:
*Ceterum censeo*: We should replace `@text` with Helix' `@markup` hierarchy: https://docs.helix-editor.com/themes.html#syntax-highlighting

Reply:
It references a specific location in a file normally, e.g.
```
#: gnulib-lib/getopt.c:345
```

Reply:
I know.

Reply:
> I know.

I think `@string.special` makes more sense due to the part after the file path doesn't really fit under `@text.uri`, but I'm open to adding both

Reply:
Well, it doesn't really fit `@string.special` either, but it's fine for now.

Maybe one day we'll rework all the highlight groups (including `@text` -> `@markup`).

Reply:
Mmm yeah I agree, let's rework this when that happens